test/gateway/anchor/anchor_state_root.js

```diff
@@ -28,171 +28,171 @@ const EventDecoder = require('../../test_lib/event_decoder.js');
 const zeroBytes =
   "0x0000000000000000000000000000000000000000000000000000000000000000";
 
-contract('SafeCore.commitStateRoot()', function (accounts) {
-
+contract('Anchor.anchorStateRoot()', function (accounts) {
+  
   let remoteChainId,
     blockHeight,
     stateRoot,
-    maxNumberOfStateRoots,
     membersManager,
-    safeCore,
+    anchor,
     owner,
-    worker;
-
+    worker,
+    maxNumberOfStateRoots;
+  
   beforeEach(async function () {
-
+    
     owner = accounts[2];
     worker = accounts[3];
     remoteChainId = new BN(1410);
     blockHeight = new BN(5);
     stateRoot = web3.utils.sha3("dummy_state_root");
     maxNumberOfStateRoots = new BN(10);
     membersManager = await MockMembersManager.new(owner, worker);
-
-    safeCore = await SafeCore.new(
+    
+    anchor = await Anchor.new(
       remoteChainId,
       blockHeight,
       stateRoot,
       maxNumberOfStateRoots,
       membersManager.address,
     );
-
+    
     stateRoot = web3.utils.sha3("dummy_state_root_1");
-
+    
   });
-
+  
   it('should fail when state root is zero', async () => {
-
+    
     stateRoot = zeroBytes;
     blockHeight = blockHeight.addn(1);
-
+    
     await Utils.expectRevert(
-      safeCore.commitStateRoot(
+      anchor.anchorStateRoot(
         blockHeight,
         stateRoot,
-        { from: worker },
+        {from: worker},
       ),
       'State root must not be zero.',
     );
-
+    
   });
-
-  it('should fail when block height is less than the latest committed ' +
+  
+  it('should fail when block height is less than the latest anchored ' +
     'state root\'s block height', async () => {
-
-      blockHeight = blockHeight.subn(1);
-
-      await Utils.expectRevert(
-        safeCore.commitStateRoot(
-          blockHeight,
-          stateRoot,
-          { from: worker },
-        ),
-        'Given block height is lower or equal to highest committed state root block height.',
-      );
-
-    });
-
-  it('should fail when block height is equal to the latest committed ' +
+    
+    blockHeight = blockHeight.subn(1);
```

Comment:
indentation was correct before, is now wrong.

Reply:
Still not fixed

Reply:
Still not fixed

Reply:
It is auto done by IDE.

Reply:
OK 👍